docs/dpdk/pod_testpmd_virt.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: testpmd
+#  annotations:
```

Comment:
no need for the commented annotation IMO

there are usecases where device plugin is used without a CNI so while sriov cni needs device plugin to operate in a k8s environment the other way is not true.